usaspending_api/agency/v2/views/agency_base.py

```diff
@@ -22,7 +22,11 @@ def toptier_code(self):
 
     @cached_property
     def fiscal_year(self):
-        fiscal_year = str(self.request.query_params.get("fiscal_year", current_fiscal_year()))
+        if self.request.method == "GET":
```

Comment:
Since the method is a GET for List and Count this can be put back to how it was.